CC/Sounder/include/RadioUHD.h

```diff
@@ -0,0 +1,47 @@
+/** @file RadioUHD.h
+  * @brief Declaration file for the RadioUHD class.
+  *
+  * Copyright (c) 2018-2022, Rice University
+  * RENEW OPEN SOURCE LICENSE: http://renew-wireless.org/license
+*/
+#ifndef RADIOUHD_H_
+#define RADIOUHD_H_
+
+#include <cstdlib>
+#include <vector>
+
+#include "SoapySDR/Device.hpp"
+#include "SoapySDR/Errors.hpp"
```

Comment:
Soapy includes necessary?

Reply:
still necessary, some "time variable" is still calculated based on the Soapy method